lib/backend.js

```diff
@@ -1,6 +1,6 @@
 import { errors } from './errors.js'
 import { errorFields, entries } from './types.js'
-import { toString } from './deno.js'
+import {decode, toString} from './deno.js'
```

Comment:
Inconsistent formatting
```suggestion
import { decode, toString } from './deno.js'
```